Adafruit_BluefruitLE_SPI.cpp

```diff
@@ -590,6 +622,7 @@ bool Adafruit_BluefruitLE_SPI::getPacket(sdepMsgResponse_t* p_response)
     spixfer(p_response->payload, p_header->length);
 
     result = true;
+    break; // Uh, no break before? Seriously..how did this work at all?
```

Comment:
Yeah, it is kind of overkill for this particular code. It is used to address several break and different handling for different error code:(